src/Server.php

```diff
@@ -549,17 +573,17 @@ protected function getSmdMethods()
     /**
      * Internal method for handling request.
      *
-     * @return void
+     * @return Error
```

Comment:
Inconsistent with declared RTH on the method.
But as noted before, it should be removed completely as no additional information provided.

Reply:
Fixed PHPDocs for now. Removal of unnecessary PHPDocs will be addressed separately.